frontends/web/src/components/rates/rates.tsx

```diff
@@ -138,8 +143,8 @@ function Conversion({
     );
   }
   return (
-    <span className={style.rates}>
-      {children}
+    <span className={ `${style.rates} ${formattedValue !== unavailableValue ? typeClassName : ''}`}>
+      {formattedValue !== unavailableValue ? sign : ''}
```

Comment:
This inline condition is a bit hard to read. Could change `const isNotAvailable = formattedValue !== unavailableValue;` and just use `isNotAvailable` here.

Reply:
Done :+1: